Dockerfile

```diff
@@ -125,15 +130,15 @@ RUN npm install --global phantomas phantomjs-prebuilt \
 ###
 RUN pip install --upgrade \
     git+https://github.com/dhs-ncats/pshtt.git@develop \
-    git+https://github.com/dhs-ncats/trustymail.git@develop
+    git+https://github.com/dhs-ncats/trustymail.git@bugfix/problem_with_psl_caching
```

Comment:
It looks like this was merged into `develop` already: https://github.com/dhs-ncats/trustymail/pull/72

Should this line get reverted back to `develop`?

Reply:
Yes, that wasn't intended to be committed.  Thanks for catching that!

Reply:
@konklone, it looks like you'll have to un-request changes before I can merge.